pages/home.jsx

```diff
@@ -25,20 +166,50 @@ var HomePage = React.createClass({
       <div>
         <HeroUnit image="/img/pages/home/hero-unit.png"
                   image2x="/img/pages/home/hero-unit@2x.png">
-          <h1>Unlock opportunities for all citizens of the Web.</h1>
+          <h1>The Mozilla Learning Network</h1>
 
         </HeroUnit>
         <div className="inner-container">
-          <div className="values">
+          <div className="about-us">
             <Illustration
-            height={226} width={226}
-            className="img-circle"
-            src1x="/img/pages/home/values.jpg" src2x="/img/pages/home/values.jpg"
-            alt="Image reflecting our values">
-              Join our community of educators and activists who want to teach digital skills and web literacy through making. <Link to="about" className="bold-link">Learn More</Link>
+              height={200} width={200}
+              src1x="/img/pages/about/about-illustration.svg" src2x="/img/pages/about/about-illustration.svg"
+              alt="">
+                <h2>About Us</h2>
+                <p>We want more people to see themselves as citizens of the web. Mozilla Learning Networks offers programs and a global community dedicated to helping people learn the most important skills of our age: <em>the ability to read, write and participate in the digital world.</em> <Link to="about" className="more">Learn more</Link></p>
             </Illustration>
           </div>
-          <CaseStudies/>
+          <BlogSection/>
+        </div>
+        <div className="quote">
+          <div className="inner-container">
+            <CaseStudies/>
+          </div>
+        </div>
+        <div className="inner-container">
+          <IconLinks>
+            <IconLink
+              href="https://twitter.com/webmaker"
+              imgSrc="/img/pages/about/svg/icon-twitter-blue.svg"
+              imgAlt="twitter logo"
+              head="Follow Us"
+              subhead="Start a conversation on Twitter"
+            />
+            <IconLink
+              href="mailto:teachtheweb@mozillafoundation.org"
+              imgSrc="/img/pages/about/svg/icon-get-help-blue.svg"
+              imgAlt="question mark icon"
+              head="Get Help"
+              subhead="Email us anytime"
+            />
+            <IconLink
+              href="http://discourse.webmaker.org/category/meet"
+              imgSrc="/img/pages/about/svg/icon-connect-blue.svg"
+              imgAlt="illustration of two people connecting"
```

Comment:
We'll be removing `imgAlt` in #906 because the icons are purely decorative, but in the meantime it might be a good idea to have `imgAlt=""` for any new code we're adding. What do you think?